examples/esp32/README.md

```diff
@@ -16,3 +16,8 @@ A small ESP-IDF app for the ESP32 using the libmicrofido2.
 
 1. Connect the ESP32 to your computer and find its device node at `/dev/ttyUSBX` (note the value of `X` and replace it in the following command).
 1. Run `docker run --rm -v $PWD/../../:/project -w /project/examples/esp32 --device /dev/ttyUSBX espressif/idf idf.py flash`.
+
+## Hardware Cryptography
+
+By default, hardware accelerated AES, partially AES-GCM, SHA-256, and SHA-512 are enabled.
+To disable these, set `CONFIG_MBEDTLS_HARDWARE_AES=n` in your `sdkconfig`.
```

Comment:
Instead, you could provide a `Kconfig.projbuild` file in the main component with the following content:
menu "FIDO configuration"

```text
config USE_HW_CRYPTO
	bool "Use hardware cryptography acceleration"
	default n
	help
		Decides whether FIDO operations will use hardware cryptography (if possible)
endmenu
```

Then you'll be able to use the `CONFIG_USE_HW_CRYPTO` variable. It will also be defined in `sdkconfig.h`.